bin/gpo

```diff
@@ -935,6 +935,9 @@ class gPodderCli(object):
         def _not_applicable(*args, **kwargs):
             pass
 
+        def _mount_volume(*args, **kwargs):
```

Comment:
this should be improved to also mount in gpo, no?

Reply:
maybe passing None as the Gio.MountOperation? Or implementing a Gio.MountOperation logging needed params?

Reply:
What happens in Gtk is that when the mount operation is handled, if additional information is needed then it shows a dialog requesting username, password, truecrypt key, etc if necessary. This dialog is built into Gtk and doesn't need to be user implemented -- for the command line it looks like I'd need to implement something what's in gvfs-mount where it figures out the needed info in the mount callback and requests it:
https://gitlab.gnome.org/GNOME/glib/-/blob/master/gio/gio-tool-mount.c
It's also possible to skip authentication which I assume would fail the mount.  I can do this if you think it's useful, I figured given gpo is likely for scripting people would probably just call gvfs-mount with the relevant parameters before gpo. That said, I'm not really sure how people do use it.

Reply:
It's also used for accessibility reasons or by users prefering the command-line.
Can you try passing `None` instead of `Gtk.MountOperation()` and see if MTP mounting works? If so we should implement this in gpo since it's useful to users.

Reply:
Yes, that works, the device gets mounted if it starts unmounted. I have added code to run the mount before sync for the CLI as well, and also fixed the hang if opening the device fails (this already exists -- it waits for done_callback which never happens, however it's possible this couldn't be triggered before with an fs sync).

As a side note I did run into what looks like https://bugs.gentoo.org/788133 along the way where starting gpodder, unmounting the device then running a sync mounts the volume but throws that error on the first fs operation.  A subsequent sync then succeeds.